cmake/Modules/opm-upscaling-prereqs.cmake

```diff
@@ -27,5 +27,6 @@ set (opm-upscaling_DEPS
 	dune-grid REQUIRED;
 	opm-common REQUIRED;
 	opm-core REQUIRED;
+        opm-output REQUIRED;
 	dune-cornerpoint REQUIRED"
```

Comment:
> Did you mean output does not require cornerpoint?

Yes...

I do not think output requires grids, why? It certainly does not in its present state.